src/components/Bag.jsx

```diff
@@ -0,0 +1,131 @@
+import { useDispatch, useSelector } from "react-redux";
+import { Flex, Popover, Text, Button } from "@sparrowengg/twigs-react";
+import { removeFromCart, setTotalAmount } from "../redux/actions";
+import { useNavigate } from "react-router-dom";
+import { useEffect, useMemo } from "react";
+const Bag = () => {
+  const cart = useSelector((state) => state.cart);
+  const totalAmount = useSelector((state) => state.totalAmount);
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    let tot = 0;
+    cart.forEach((ele) => {
+      tot = tot + Number(ele.cartCount) * Number(ele.price);
+    });
+    tot = Number(tot.toFixed(3));
+    dispatch(setTotalAmount(tot));
+  }, [cart]);
+
+  // const bagTotal = useMemo(() => {
+  //   let tot = 0;
+  //   cart.forEach((ele) => {
+  //     tot = tot + Number(ele.cartCount) * Number(ele.price);
+  //   });
+  //   return tot.toFixed(2);
+  // }, [cart]);
+  return (
+    <>
+      <Flex
+        flexDirection="column"
+        justifyContent="space-between"
+        gap="10px"
```

Comment:
@arun-121 we have our theming values for example 10px is considered as $5, can you check the entire pr and here is the documentation for reference https://twigs.surveysparrow.com/docs/theming

Reply:
@arun-121 spaces like margin, gap and padding ranges from $1 to $50, which each values is multiples of two for example $4 -> 8px,

for height and width it is multiples of four for height and width -> $4 -> 16px

Reply:
used theming variables